### PR TITLE
Classification Metrics & Updating Some Regression Metrics Where Return Type Is Not Fixed

### PR DESCRIPTION
**Pull request by harshil2493**
Adding Following Metrics 

Cohen Kappa
F1
FBeta
Hamming Loss
Hinge Loss
Jaccard Similarity Score
Log Loss
Matthews Correlation Coefficient
Precision Recall Curve
✘ Precision Recall Fscore Support
Precision Score
Recall Score
ROC AUC Score
ROC Curve
Zero One Loss

Updating Following Where Functions Does Not Only Return Single Value, But Rather Array Of Values Too Depends On Param Provided

Regression
Explained Variance
Mean Absolute Error
Mean Squared Error
Mean Squared Log Error
R2 Score

### CODE REVIEW

File: mlops/parallelm/mlops/ml_metrics_stat/classification/classification_stat_object_factory.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def get_mlops_cohen_kappa_score_stat_object(**kwargs):
         Method will create MLOps Single value stat object from numeric real number - cohen kappa score
         It is not recommended to access this method without understanding single value data structure that it is returning.
         :param kwargs: numeric value of cohen kappa score
-        :return: Single Value stat object which has brier score embedded inside
+        :return: Single Value stat object which has cohen kappa score embedded inside
```
Comment (yakov-g) [inline mlops/parallelm/mlops/ml_metrics_stat/classification/classification_stat_object_factory.py]: this is commit for F1 score, not cohen kappa

File: mlops/parallelm/mlops/ml_metrics_stat/regression/regression_stat_object_factory.py [hunk 2]
```diff
@@ -35,18 +35,26 @@ def get_mlops_explained_variance_score_stat_object(**kwargs):
     @staticmethod
     def get_mlops_mean_absolute_error_stat_object(**kwargs):
         """
-        Method will create MLOps Single value stat object from numeric real number - mean absolute error
-        It is not recommended to access this method without understanding single value data structure that it is returning.
-        :param kwargs: mean absolute error
-        :return: Single Value stat object which has mean absolute error embedded inside
+        Method will create MLOps Single/Multiline value stat object from numeric real number - mean absolute error; or list of errors
```
Comment (yakov-g) [inline mlops/parallelm/mlops/ml_metrics_stat/regression/regression_stat_object_factory.py]: When I used will, @DrewParallel said present should be used: Method creates

File: mlops/tests/test_classification_metrics.py [hunk 3]
```diff
@@ -626,3 +626,36 @@ def test_mlops_recall_score_apis():
     assert len(recall_score) == 3
 
     pm.done()
+
+
```
Comment (yakov-g) [inline mlops/tests/test_classification_metrics.py:630]: most likely IntelliJ tells you that there should be 1 line interval

**Review by yakov-g — APPROVED** [review]
Small fixes

File: mlops/parallelm/mlops/ml_metrics_stat/regression/regression_stat_object_factory.py (hunk 2 above)
Comment (DrewParallel) [reply mlops/parallelm/mlops/ml_metrics_stat/regression/regression_stat_object_factory.py]: Yup

File: mlops/tests/test_classification_metrics.py (hunk 3 above)
Comment (harshilpm) [reply mlops/tests/test_classification_metrics.py:630]: it does!

**Review by sriram109 — APPROVED** [review]

**Pull request merged**